README.md

```diff
@@ -51,10 +55,13 @@ for all the passed options.
 ## Options
 
   - `cache`           Compiled functions are cached, requires `filename`
-  - `filename`        Used by `cache` to key caches, and for includes
+  - `filename`        The name of the file being rendered. Not required if you 
```

Comment:
I'm not sure that Markdown renderers do the right thing with line continuations like this. I believe this is why the lines here were so long. Can we verify this actually looks right when rendered?